Modules/Loadable/Segmentations/EditorEffects/Python/AbstractScriptedSegmentEditorAutoCompleteEffect.py

```diff
@@ -293,8 +293,14 @@ def onApply(self):
       segmentID = segmentIDs.GetValue(index)
       previewSegment = previewNode.GetSegmentation().GetSegment(segmentID)
       previewSegmentLabelmap = previewSegment.GetRepresentation(vtkSegmentationCore.vtkSegmentationConverter.GetSegmentationBinaryLabelmapRepresentationName())
-      self.scriptedEffect.modifySegmentByLabelmap(segmentationNode, segmentID,previewSegmentLabelmap,
+
+      oldOverwriteMode = self.scriptedEffect.parameterSetNode().GetOverwriteMode()
+      self.scriptedEffect.parameterSetNode().SetOverwriteMode(slicer.vtkMRMLSegmentEditorNode.OverwriteAllSegments)
```

Comment:
Why is this necessary?

Auto-complete effect ignores hidden segments, but now with this change, those segments would be modified, which is probably not what the user wants.